scripts/run_pfioh_pull

```diff
@@ -6,7 +6,7 @@ pfurl --verb POST --raw --httpResponseBodyParse --http $1/api/v1/cmd --msg \
             \"key\":         \"$2\"
```

Comment:
you could just use the pfurl client code for each of these scripts.  Would be cleaner.

Reply:
I researched the pfurl repository in PyPI, but its documentation led me to github with bash scripts. So, I don't know the specific usage of the pfurl client code.

Reply:
There are examples in pfcon:

https://github.com/FNNDSC/pfcon/blob/master/pfcon/pfcon.py#L514

https://github.com/FNNDSC/pfcon/blob/master/pfcon/pfcon.py#L434